x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/FullClusterRestartIT.java

```diff
@@ -633,7 +633,12 @@ private void assertRollUpJob(final String rollupJob) throws Exception {
         waitForRollUpJob(rollupJob, expectedStates);
 
         // check that the rollup job is started using the RollUp API
-        final Request getRollupJobRequest = new Request("GET", "_xpack/rollup/job/" + rollupJob);
+        final Request getRollupJobRequest;
+        if (isRunningAgainstOldCluster()) {
+            getRollupJobRequest = new Request("GET", "/_xpack/rollup/job/" + rollupJob);
```

Comment:
These are required because we can do a full cluster restart against versions that don't have the new endpoint.